### PR TITLE
MOBP-8034 || Add a parameter to always zoom and center

### PR DESCRIPTION
**Pull request by przanBrainly**
https://brainly.atlassian.net/browse/MOBP-8034

Add a parameter to always center and zoom crop view when user lifts finger.

https://github.com/brainly/Android-Image-Cropper/assets/108520654/1d1a373f-e841-4472-95c4-cd619e6bdd0a

The issue with crop label near the top of the screen will be resolved in another PR.

https://github.com/brainly/Android-Image-Cropper/assets/108520654/36d12744-8f60-4ee7-ab3d-6e79ee0c53c1

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -1,79 +1,129 @@
-#!/usr/bin/env bash
+#!/bin/sh
```
Comment (arkadiuszkaczorowski) [inline gradlew:1]: just wonder why the whole file is changed? Can we avoid this?

File: gradlew.bat [hunk 2]
```diff
@@ -1,3 +1,19 @@
+@rem
```
Comment (arkadiuszkaczorowski) [inline gradlew.bat:1]: ust wonder why the whole file is changed? Can we avoid this?

Comment (przanBrainly) [reply gradlew.bat:1]: I couldn't build the project as it required old java to be installed. Android Studio suggested to update gradle which I did. This code was modified by Android Studio, I haven't touched it. Can it stay or you have other ideas?

Comment (arkadiuszkaczorowski) [reply gradlew.bat:1]: It can stay, just wondered why this is needed - it it is a must then it is fine, thx 💪

**Review by arkadiuszkaczorowski — APPROVED** [review]
Awesome ✨

**Review by admund — APPROVED** [review]

**Pull request merged**